x/group/types.go

```diff
@@ -118,6 +113,84 @@ func (p *ThresholdDecisionPolicy) Validate(g GroupInfo) error {
 	return nil
 }
 
+// Implements DecisionPolicy Interface
+var _ DecisionPolicy = &PercentageDecisionPolicy{}
+
+// NewPercentageDecisionPolicy creates a new percentage DecisionPolicy
+func NewPercentageDecisionPolicy(percentage string, timeout time.Duration) DecisionPolicy {
+	return &PercentageDecisionPolicy{percentage, timeout}
+}
+
+func (p PercentageDecisionPolicy) ValidateBasic() error {
+	percentage, err := math.NewPositiveDecFromString(p.Percentage)
+	if err != nil {
+		return sdkerrors.Wrap(err, "percentage threshold")
+	}
+	value, err := percentage.Float64()
+	if err != nil {
+		return sdkerrors.Wrap(err, "Dec to Float64 conversion of percentage")
+	}
+	if !(value >= 0 && value <= 1) {
```

Comment:
I think we shouldn't allow 0 value
which is actually already enforced by `NewPositiveDecFromString ` anyway, so I guess here you safely just have to check that `value > 1`
you could also just compare it using `Cmp` instead of adding `Float64()` method